src/patternfly/components/Button/button.scss

```diff
@@ -294,6 +301,9 @@
   border-start-end-radius: var(--#{$button}--BorderStartEndRadius, var(--#{$button}--BorderRadius));
   border-end-start-radius: var(--#{$button}--BorderEndStartRadius, var(--#{$button}--BorderRadius));
   border-end-end-radius: var(--#{$button}--BorderEndEndRadius, var(--#{$button}--BorderRadius));
+  transition-timing-function: var(--#{$button}--TransitionTimingFunction);
+  transition-duration: var(--#{$button}--TransitionDuration);
+  transition-property: background-color;
```

Comment:
Just a nit and you may know this but the shorthand version would be

```
transition: background-color var(--#{$button}--TransitionDuration) var(--#{$button}--TransitionTimingFunction);
```

Reply:
Since the property isn't a variable I wasn't sure if the shorthand version was advisable or not? I can change it if you think it's better.

Reply:
@mcoker I realized color needed to be transformed as well, so I left this as is since the shorthand doesn't take multiple properties (unless we use `all`)